offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectPresenter.groovy

```diff
@@ -4,8 +4,11 @@ import life.qbic.business.projects.create.CreateProjectOutput
 import life.qbic.datamodel.dtos.business.OfferId
 import life.qbic.datamodel.dtos.projectmanagement.Project
 import life.qbic.datamodel.dtos.projectmanagement.ProjectIdentifier
+import life.qbic.portal.offermanager.communication.EventEmitter
 import life.qbic.portal.offermanager.components.AppViewModel
 
+import java.awt.Event
+
```

Comment:
?

Reply:
Auto import, happens to me all the time when I type `new EventEmitter` 🤦

Reply:
good catch 👍🏼

Reply:
Thanks the auto implement happened to me to sometimes. I noticed it in the GitHub notification and thought I would raise awareness :D